src/connector/postgres.rs

```diff
@@ -226,6 +236,7 @@ impl PostgresUrl {
         let mut socket_timeout = None;
         let mut connect_timeout = None;
         let mut pg_bouncer = false;
+        let mut statement_cache_size = 0;
```

Comment:
We'll probably figure things out as we start using it, but for other quaint users, if it works we may want to default to a non-zero size.

Reply:
I don't want to enable this to everybody by default. Should be opt-in.

Reply:
There's no other way to reuse prepared statements currently, so this should at least to be well documented.

Reply:
Ok. Making the default here to 500. Let's see...